tbtc-ts/src/index.ts

```diff
@@ -76,6 +78,40 @@ export interface TBTC {
 
   // TODO: Implementation and documentation.
   revealDeposit(): Promise<void>
+
+  /**
+   * Creates a Bitcoin P2WPKH sweep transaction.
```

Comment:
We probably want to keep it P2WPKH eventually but given our current struggle with P2SH and P2WSH, I just keep a note here this can be changed in the future.